test/parallel/test-https-byteswritten.js

```diff
@@ -29,8 +30,8 @@ const fs = require('fs');
 const https = require('https');
 
 const options = {
-  key: fs.readFileSync(`${common.fixturesDir}/keys/agent1-key.pem`),
-  cert: fs.readFileSync(`${common.fixturesDir}/keys/agent1-cert.pem`)
+  key: fs.readFileSync(`${fixtures.fixturesDir}/keys/agent1-key.pem`),
+  cert: fs.readFileSync(`${fixtures.fixturesDir}/keys/agent1-cert.pem`)
```

Comment:
There is the dedicated `fixtures.readkey` function that could and should be used instead. Would you be so kind and please update this?